Sdk/Speckle.Connectors.Utils/ContainerRegistration.cs

```diff
@@ -15,6 +15,7 @@ public static void AddConnectorUtils(this SpeckleContainerBuilder builder)
     // send operation and dependencies
     builder.AddSingleton<CancellationManager>();
     builder.AddScoped<ReceiveOperation>();
+    builder.AddScoped<AccountService>();
```

Comment:
from usage, to me it looks like this dude could be better as a singleton?

Reply:
(not essential)

Reply:
yea, doing it